app/src/main/java/fr/free/nrw/commons/feedback/FeedbackContentCreator.java

```diff
@@ -0,0 +1,77 @@
+package fr.free.nrw.commons.feedback;
+
+import android.content.Context;
+import fr.free.nrw.commons.R;
+import fr.free.nrw.commons.feedback.model.Feedback;
+import fr.free.nrw.commons.utils.LangCodeUtils;
+import java.util.Locale;
+
+/**
+ * Creates a wikimedia recognizable format
+ * from feedback information
+ */
+public class FeedbackContentCreator {
+    private StringBuilder stringBuilder;
+    private Feedback feedback;
+    private Context context;
+
+    public FeedbackContentCreator(Context context, Feedback feedback) {
+        this.feedback = feedback;
+        this.context = context;
+        init();
+    }
+
+    /**
+     * Initializes the string buffer object to append content from feedback object
+     */
+    public void init() {
+        // Localization is not needed here, because this ends up on a page where developers read the feedback, so English is the most convenient.
+
+        stringBuilder = new StringBuilder();
+        stringBuilder.append("== ").append(LangCodeUtils.getLocalizedResources(context,
```

Comment:
From a readability point of view, I would write:
```
stringBuilder.append("== ");
stringBuilder.append("Feedback for version ");
stringBuilder.append(feedback.getVersion());
stringBuilder.append(" ==");
```
(about the second line see review comment below)